Add tournament `Spanish Showdown 2026` (#14695)
ppy/osu-wiki: wiki/Tournaments/SS/2026/en.md

---
tags:
  - SS26
  - SS
---

# Spanish Showdown 2026

> ![banner](https://methalox.s-ul.eu/QoTGDhpU)

Reviewer: place this image in an `img/` folder beside the `en.md` file, not on an external host
 
```suggestion
![Spanish Showdown 2026 banner](img/banner.jpg)
```

Spanish Showdown 2026 (SS26) is the most well known spanish-only 1v1 open rank tournament, featuring low SR pools at lower stages for new players and a very high skill ceiling in later stages. Only the best will prevail.

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2026-02-02/2026-02-15 |
| Qualifiers showcase | 2026-02-15 |
| Qualifiers stage | 2026-02-16/2026-03-01 |
| Seed reveal & Group draw | 2026-03-01 |
| Group Stage | 2026-03-01/2026-03-15 |
| Round of 32 | 2026-03-16/2026-03-22 |
| Round of 16 | 2026-03-23/2026-03-29 |
| Quarterfinals | 2026-03-30/2026-04-05 |
| Semifinals | 2026-04-06/2026-04-12 |
| Finals | 2026-04-13/2026-04-19 |
| Grand Finals | 2026-04-19/2026-04-26 |

## Prizes

> This year's iteration offered 482.15€ and a total of 2y of supporter thanks to the donations made by several members of the community 

Reviewer: ```suggestion
This year's iteration offered €482.15 and a total of 2 years of osu!supporter thanks to the donations made by several members of the community.
```

| Placing | Prizes |
| :-: | :-- |

> | ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 46.45% of the prize pool, profile badge, 1y of osu!supporter |

Reviewer: ```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 46.45% of the prize pool, profile badge, 1 year of osu!supporter |
```

| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 25.49% of the prize pool, 6 months of osu!supporter |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 13.99% of the prize pool, 6 months of osu!supporter |
| 4th | 7.68% of the prize pool |
| 5th, 6th | 2.105% of the prize pool |

> | 7th, 8th | 1.155% of the prize pool |

Reviewer: why 3 decimal places here but not above?

> | 7th, 8th | 1.155% of the prize pool |

Reviewer: ```suggestion
| 5–6th | 2.105% of the prize pool |
| 7–8th | 1.155% of the prize pool |
```

> ![](https://aleph-003.s-ul.eu/G6SXBB59)

Reviewer: again, move this file into the repository

```suggestion
![Spanish Showdown 2026 winner's badge](img/badge.png "Spanish Showdown 2026 winner's badge")
```

## Organisation

| Position | Member(s) |
| :-- | :-- |

> | Hosts | ::{ flag=ES }:: Furina, ::{ flag=ES }:: jdfs, ::{ flag=ES }:: Hide in room |

Reviewer: missing user links

| Custom mappers | ::{ flag=ES }:: Yuri Enjoyer, ::{ flag=ES }:: Jurumas, ::{ flag=AR }:: -Links, ::{ flag=ES }:: Radiownd, ::{ flag=ES }:: Straweeeee, ::{ flag=ES }:: Sakorii, ::{ flag=ES }:: Akareh, ::{ flag=ES }:: Marcobrolo, ::{ flag=ES }:: Hide in room |
| Playtesting | ::{ flag=ES }:: Hide in room, ::{ flag=GR }:: -Puyu, ::{ flag=RU }:: Endura, ::{ flag=ES }:: CharlioM9, ::{ flag=ES }:: BabyScylla, ::{ flag=ES }:: Scaryz, ::{ flag=ES }:: Adronks, ::{ flag=ES }:: creator, ::{ flag=ES }:: \[dekori\], ::{ flag=ES }:: marbl |
| Referees | ::{ flag=ES }:: Furina, ::{ flag=ES }:: jdfs, ::{ flag=ES }:: junamat, ::{ flag=ES }:: Hide in room |
| Casters | ::{ flag=ES }:: Hide in room, ::{ flag=ES }:: Scaryz, ::{ flag=ES }:: Diesel, ::{ flag=ES }:: EDn, ::{ flag=ES }:: towny1, ::{ flag=ES }:: senya-, ::{ flag=ES }:: ESCRUPULILLO, ::{ flag=ES }:: Jurumas, ::{ flag=ES }:: Furina, ::{ flag=ES }:: Dario, ::{ flag=ES }:: marbl, ::{ flag=ES }:: A L E P H, ::{ flag=ES }:: Kalal331, ::{ flag=ES }:: junamat |
| Streaming | ::{ flag=ES }:: jdfs, ::{ flag=ES }:: Furina, ::{ flag=ES }:: marbl, ::{ flag=ES }:: Hide in room |

> | Graphics & Frontend | ::{ flag=ES }:: Yuri Enjoyer, ::{ flag=ES }:: Agatem |

Reviewer: ```suggestion
| Graphics & frontend | ::{ flag=ES }:: Yuri Enjoyer, ::{ flag=ES }:: Agatem |
```

| Backend Dev | ::{ flag=ES }:: Furina, ::{ flag=ES }:: jdfs, ::{ flag=ES }:: junamat |

> | Lazer Dev | ::{ flag=ES }:: Furina |

Reviewer: ```suggestion
| Backend developer | ::{ flag=ES }:: Furina, ::{ flag=ES }:: jdfs, ::{ flag=ES }:: junamat |
| Lazer developer | ::{ flag=ES }:: Furina |
```

| Liquipedia page | ::{ flag=RU }:: ThatAvocado_Boi |

## Links

### About current iteration

- [Web](https://www.spanishshowdown.es)
- [Twitch](https://twitch.tv/SpanishShowdown)
- [Discord](https://discord.gg/cQ2j2Qu4Kb)
- [Bracket](https://challonge.com/SS2026)
- [Youtube](https://youtube.com/@SpanishShowdown)
- [Liquipedia](https://liquipedia.net/osu/Spanish_Showdown/2026)
- [Forum post](https://osu.ppy.sh/community/forums/topics/2176155)

> - [Fallback sheet](https://docs.google.com/spreadsheets/d/1FgbuKpRj0HYLiZOZQOS04q10KKCR18MGRTgaCbD3xCA/edit?gid=1648385434#gid=1648385434)

Reviewer: ```suggestion
- [Website](https://www.spanishshowdown.es)
- [Twitch channel](https://twitch.tv/SpanishShowdown)
- [Discord server](https://discord.com/invite/cQ2j2Qu4Kb)
- [Challonge bracket](https://challonge.com/SS2026)
- [YouTube channel](https://youtube.com/@SpanishShowdown)
- [Liquipedia page](https://liquipedia.net/osu/Spanish_Showdown/2026)
- [Forum post](https://osu.ppy.sh/community/forums/topics/2176155)
- [Fallback spreadsheet](https://docs.google.com/spreadsheets/d/1FgbuKpRj0HYLiZOZQOS04q10KKCR18MGRTgaCbD3xCA/edit?gid=1648385434#gid=1648385434)
```

### Last iterations

- [Spanish Showdown 2024](https://2024.spanishshowdown.es) 

> - [Spanish Showdown 2025](https://2025.spanishshowdown.es)

Reviewer: these should be linked in respective articles about the tournaments in question, not here

```suggestion
```

## Participants

> - ::{ flag=ES }:: PEAC 3

Reviewer: missing user links

- ::{ flag=ES }:: PEAC 4
- ::{ flag=ES }:: Jairofrancaso
- ::{ flag=ES }:: Rumpowl
- ::{ flag=ES }:: Fieera
- ::{ flag=ES }:: \-Amadeus\-
- ::{ flag=ES }:: Edrran
- ::{ flag=ES }:: Pinwi_
- ::{ flag=ES }:: Seto
- ::{ flag=ES }:: FestivalFour179
- ::{ flag=ES }:: EDn
- ::{ flag=ES }:: Krla
- ::{ flag=ES }:: sandro114
- ::{ flag=ES }:: galaxyxy
- ::{ flag=ES }:: A N E R
- ::{ flag=ES }:: LaBestiaPrecoz
- ::{ flag=ES }:: Murkano
- ::{ flag=ES }:: Dario
- ::{ flag=ES }:: Kalal331
- ::{ flag=ES }:: ESCRUPULILLO
- ::{ flag=ES }:: Feren
- ::{ flag=ES }:: Paraca405
- ::{ flag=ES }:: NHarmonia
- ::{ flag=ES }:: comandocYT
- ::{ flag=ES }:: Widelux13
- ::{ flag=ES }:: n0ught
- ::{ flag=ES }:: MeiFTW
- ::{ flag=ES }:: Omeguii
- ::{ flag=ES }:: \-aitor\-
- ::{ flag=ES }:: AARONGAMER1
- ::{ flag=ES }:: leo24012007
- ::{ flag=ES }:: atisuma
- ::{ flag=ES }:: \[Ajoduro\]
- ::{ flag=ES }:: \- Izayoi \-
- ::{ flag=ES }:: Zaik0
- ::{ flag=ES }:: Palodex
- ::{ flag=ES }:: Scaryz
- ::{ flag=ES }:: JoanaBob
- ::{ flag=ES }:: \[dekori\]
- ::{ flag=ES }:: EZHD
- ::{ flag=ES }:: PEAC 10
- ::{ flag=ES }:: amborgesa
- ::{ flag=ES }:: P_F_C_PRO

## Podium

| Placing | Name |
| :-: | :-- |

> | ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=ES }:: A L E P H |

Reviewer: missing user links

| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=ES }:: towny1 |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=ES }:: ESCRUPULILLO |

## Mappools

### Qualifiers

**[Download the mappack here](https://drive.google.com/file/d/1kIngmO5VDQRgyo1hYO1s3ZnvS0lFgtRX/view?usp=sharing)**
**[View the showcase VOD here](https://www.youtube.com/watch?v=0OJZtsKK4hs)**

> - NoMod

Reviewer: add space between "No Mod", "Hard Rock", "Double Time", etc

  4. [KING & Princess - Drastic Dramatic [Extreme]](https://osu.ppy.sh/beatmapsets/2206803#osu/4672803)
- HardRock
  1. [Camellia - Kono Hoshi de.... (20th Anniversary Remak [Loli's Extreme]](https://osu.ppy.sh/beatmapsets/2287988#osu/4904162)
  2. [beatMARIO (COOL&CREATE) - Kyouki no Hitomi ~ Invisible Full Moon [alvearia's Extra]](https://osu.ppy.sh/beatmapsets/1827329#osu/4656332)
  3. [USAO - Glaring Eyes [Tech Grandpas]](https://osu.ppy.sh/beatmapsets/2538564#osu/5617887)
  4. [KOKIA - Chouwa oto ~with reflection~ (Cut Ver.) [Expert]](https://osu.ppy.sh/beatmapsets/2156509#osu/4546068)
- DoubleTime
  1. [Annabel - pedalnote [twilight]](https://osu.ppy.sh/beatmapsets/1801386#osu/3693426)
  2. [GOETIA - Siege of Tobruk [The Rats of Tobruk]](https://osu.ppy.sh/beatmapsets/2527286#osu/5583587)
  3. [Ayane - Arrival of Tears [Insane]](https://osu.ppy.sh/beatmapsets/2108157#osu/4425512)
  4. [KOTOKO - Wing my Way [Endless Sky]](https://osu.ppy.sh/beatmapsets/850548#osu/1778268)
- Tiebreaker
  1. **[katagiri - Kyuukou [Echoes of the Forgotten]](https://osu.ppy.sh/beatmapsets/1952542#osu/4043359)**

> ### Grand Finals

Reviewer: these sections are in the wrong order. place Grand Finals at the top and Qualifiers at the bottom

> **[Mappack download here](https://drive.google.com/file/d/1Y4Ss9e_bRauL1QOFbG8KsoJwfkS1EWx6/view?usp=sharing)**

Reviewer: append a backslash when stacking hatnotes

```suggestion
**[Mappack download here](https://drive.google.com/file/d/1Y4Ss9e_bRauL1QOFbG8KsoJwfkS1EWx6/view?usp=sharing)**\
```

> **[Mappack download here](https://drive.google.com/file/d/1Y4Ss9e_bRauL1QOFbG8KsoJwfkS1EWx6/view?usp=sharing)**

Reviewer: take a look at the [template](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE.md#mappools) for the wording on these

example:
<img width="1228" height="154" alt="Image" src="https://github.com/user-attachments/assets/f4790e36-f6b2-4cc6-9f78-df85225853f3" />

  3. [LV.4 - Burning Star [Burn Out]](https://osu.ppy.sh/beatmapsets/2095154#osu/4393036)
  4. [Shade - Mars [Reload]](https://osu.ppy.sh/beatmapsets/1276364#osu/2651853)
- HardRock
  1. [hasu - Warcry [THOUSAND PUNCH (kills instantly)]](https://osu.ppy.sh/beatmapsets/2281532#osu/4864391)
  2. [STEAKA - Doping Dance [Spiral]](https://osu.ppy.sh/beatmapsets/2137942#osu/4499418)
  3. [void - Just Hold on (To All Fighters) [pip & enryotoki's euphoric lieselotte]](https://osu.ppy.sh/beatmapsets/2010202#osu/4182702)
  4. [ZUN - Shigen no Beat ~ Pristine Beat [Thunder God's Anger]](https://osu.ppy.sh/beatmapsets/2021984#osu/4210974)
- DoubleTime
  1. [Wakeshima Kanon - Not For Sale Fossil [Frail]](https://osu.ppy.sh/beatmapsets/1144490#osu/2389623)
  2. [Amane - TWEEKER [Lunatic]](https://osu.ppy.sh/beatmapsets/257165#osu/771858)
  3. [BlackY feat Risa Yuzuki - HOT LIMIT Remix [Fairy dance]](https://osu.ppy.sh/beatmapsets/1732883#osu/3541966)
  4. [Minatsu Eru - Kessen [JHC's Another]](https://osu.ppy.sh/beatmapsets/2375327#osu/5253878)
- Tiebreaker
  1. **[Laur - SEV-26 [Abnormal Severe Neural Decay]](https://osu.ppy.sh/beatmapsets/2511820#osu/5558461)**

## Match results

> ### Qualifiers results

Reviewer: ```suggestion
### Qualifier results
```

Ordered by Z-Sum. For more info on how this seeding method works, check it out [here](/wiki/Tournaments/Common_seeding_methods).

Detailed statistics for this round can be found [here](https://www.spanishshowdown.es/stats/1).

| Rank | Player | Z-Sum | Avg. Score |

> | :-- | :-- | :-- | :-- |

Reviewer: table headers should be in sentence case
the column alignments could be more consistent with similar tables in the wiki

```suggestion
| Rank | Player | Z-Sum | Avg. score |
| :-: | :-- |--: | --: |
```

| #41 | ::{ flag=ES }:: MeiFTW | -6.01 | 139004 |
| #42 | ::{ flag=ES }:: Pinwi_ | -6.98 | 103117 |
| #43 | ::{ flag=ES }:: Tenjun | -7.03 | 129001 |
| #44 | ::{ flag=ES }:: n0ught | -7.29 | 113223 |
| #45 | ::{ flag=ES }:: Agatem | -7.65 | 101612 |
| #46 | ::{ flag=ES }:: FestivalFour179 | -7.67 | 95527 |
| #47 | ::{ flag=ES }:: laGmaN | -7.94 | 91547 |
| #48 | ::{ flag=ES }:: leo24012007 | -8.02 | 120071 |
| #49 | ::{ flag=ES }:: Krla | -8.24 | 90012 |
| #50 | ::{ flag=ES }:: darkmichigun | -9.75 | 60954 |
| #51 | ::{ flag=ES }:: supley | -9.83 | 60312 |
| #52 | ::{ flag=ES }:: Feren | -10.55 | 51562 |
| #53 | ::{ flag=ES }:: 500skills | -10.66 | 50002 |
| #54 | ::{ flag=ES }:: sandro114 | -10.93 | 41043 |
| #55 | ::{ flag=ES }:: - ryochi - | -11.07 | 42914 |
| #56 | ::{ flag=ES }:: Seto | -12 | 27029 |
| #57 | ::{ flag=ES }:: Erikillo | -12.38 | 18079 |

### Group Stage

Detailed statistics for this round can be found [here](https://www.spanishshowdown.es/stats/4).

Friday, 6 March 2026:

| ID | Player 1 |  |  | Player 2 | Match link | VOD link | Referee | Streamer | Casters |
| :-- | :-- | --: | --: | :-- | :-- | :-- | :-- | :-- | :-- |

> | C2 | **aleotcapitalist** ::{ flag=ES }:: | **5** | 1 | ::{ flag=ES }:: MeiFTW | [#1](https://osu.ppy.sh/community/matches/120665907) | [#1](https://www.youtube.com/watch?v=H8NX8BnYFhY) | Furina | Furina | Furina, junamat |

Reviewer: missing flags and user links

Author: should flags be added to referee, streamer & casters too?

| :-- | :-- | --: | --: | :-- | :-- | :-- | :-- | :-- | :-- |
| 57 | **[dekori]** ::{ flag=ES }:: | **0** | FF | ::{ flag=ES }:: Dario | WBD | - | - | - | - |
| 58 | **towny1** ::{ flag=ES }:: | **7** | 1 | ::{ flag=ES }:: Jairofrancaso | [#1](https://osu.ppy.sh/community/matches/120958985) | [#1](https://www.youtube.com/watch?v=9QSyb5yp3yQ) | Furina | Furina | Scaryz, A L E P H |

Saturday, 18 April 2026:

| ID | Player 1 |  |  | Player 2 | Match link | VOD link | Referee | Streamer | Casters |
| :-- | :-- | --: | --: | :-- | :-- | :-- | :-- | :-- | :-- |
| 59 | **towny1** ::{ flag=ES }:: | **7** | 6 | ::{ flag=ES }:: [dekori] | [#1](https://osu.ppy.sh/community/matches/120967175) | [#1](https://www.youtube.com/watch?v=BwMh2yAeqsc) | jdfs | jdfs | Furina, junamat |

Sunday, 19 April 2026:

| ID | Player 1 |  |  | Player 2 | Match link | VOD link | Referee | Streamer | Casters |
| :-- | :-- | --: | --: | :-- | :-- | :-- | :-- | :-- | :-- |
| 60 | ESCRUPULILLO ::{ flag=ES }:: | 1 | **7** | ::{ flag=ES }:: **A L E P H** | [#1](https://osu.ppy.sh/community/matches/120974698) | [#1](https://www.youtube.com/watch?v=EzC45eGLZwM) | jdfs | jdfs | Scaryz, junamat |

> ### Grand Finals

Reviewer: again, your sections are in the wrong order

Author: what is it that is in the wrong order? is it related to referee/streamer/caster stuff?

Author: just noticed it, solved

Detailed statistics for this round can be found [here](https://www.spanishshowdown.es/stats/10).

Saturday, 25 April 2026:

| ID | Player 1 |  |  | Player 2 | Match link | VOD link | Referee | Streamer | Casters |
| :-- | :-- | --: | --: | :-- | :-- | :-- | :-- | :-- | :-- |
| 61 | ESCRUPULILLO ::{ flag=ES }:: | 7 | **8** | ::{ flag=ES }:: **towny1** | [#1](https://osu.ppy.sh/community/matches/121012417) | [#1](https://www.youtube.com/watch?v=4Mvm3IUj4JI) | Furina | Furina | Hide in room, jdfs, Scaryz |

Sunday, 26 April 2026:

| ID | Player 1 |  |  | Player 2 | Match link | VOD link | Referee | Streamer | Casters |
| :-- | :-- | --: | --: | :-- | :-- | :-- | :-- | :-- | :-- |
| 62 | **A L E P H** ::{ flag=ES }:: | **8** | 2 | ::{ flag=ES }:: towny1 | [#1](https://osu.ppy.sh/community/matches/121019897) | [#1](https://www.youtube.com/watch?v=JSsOagiNuGo) | jdfs | Furina | Furina, Scaryz |

## Groups

> Groups were made with a draw. Players were separated in 2 different groups using qualifier results as follows:

Reviewer: this whole explanation should go in the ruleset

- First draft: Seeds 17-20, 25-28, 33-36, 41-44

> - Second draft: Seeds 21-24, 29-32, 37-40, 45-48

Reviewer: en dashes for number ranges

```suggestion
- First draft: Seeds 17–20, 25–28, 33–36, 41–44
- Second draft: Seeds 21–24, 29–32, 37–40, 45–48
```

After the group stage, the two top players from each group go to round of 32 playoffs, assigning the seeds by doing an average seed for each group. Here's how it is done for each one of them:

| Group | Avg. qualifying seed | Top 1 seed for playoffs | Top 2 seed for playoffs |
| :-- | :-: | :-: | :-: |
| A | 31.25 | 20 | 28 |
| B | 33.75 | 21 | 29 |
| C | 29.00 | 17 | 25 |
| D | 34.25 | 22 | 30 |
| E | 31.00 | 19 | 27 |
| F | 35.25 | 24 | 32 |
| G | 30.75 | 18 | 26 |
| H | 34.75 | 23 | 31 |

Watch the drawing VOD [here!](https://youtu.be/ISG7TqshQVU&t=2879) 

### Group A

| # | Player | Pts | PIF | PA | SUM |
| :-- | :-- | :-: | :-: | :-: | :-: |
| 1 | #20 Fieera | 0 | 15 | 5 | 10 |
| 2 | #28 Kalal331 | 0 | 14 | 6 | 8 |
| 3 | Tenjun | 0 | 7 | 15 | -8 |
| 4 | comandocYT | 0 | 0 | 15 | -15 |

> #### Match History

Reviewer: sentence case for headings, not title case

| Match | Red Team | Score | Blue Team | MP Link |
| :-- | :-- | :-: | :-- | :-- |
| A1 | Kalal331 | 4 - 5 | **Fieera** | [Link](https://osu.ppy.sh/community/matches/120674652) |
| A2 | comandocYT | 0 - 5 | **Tenjun** | FF |
| A3 | **Kalal331** | 5 - 0 | comandocYT | FF |
| A4 | **Fieera** | 5 - 1 | Tenjun | [Link](https://osu.ppy.sh/community/matches/120722477) |
| A5 | **Kalal331** | 5 - 1 | Tenjun | [Link](https://osu.ppy.sh/community/matches/120683156) |

> | A6 | **Fieera** | 5 - 0 | comandocYT | [Link](https://osu.ppy.sh/community/matches/120683552) |

Reviewer: please match the template https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE.md#match-results, like you've done with the other match results

2. In case of not receiving the invitation or a sudden disconnection, contact the referee assigned to the room.
3. Default lobby schedules will be available on the tournament website. If none of the lobbies suit you, you may request a new lobby in the dedicated Discord channel (#qualifiers-schedule).
4. In the room, you will play the maps in order (NoMod -> Hidden -> HardRock -> DoubleTime) (10 maps).
5. The mappool can only be played once.
6. The scoring system used will be Z-Score. Refer to [this wiki page](/wiki/Tournaments/Common_seeding_methods) in order to check out how it works.
7. Qualifiers results will remain private throughout the process; lobby links will not be public either.
8. If a player is found sniping mp links from qualifier rooms, they will be irrevocably disqualified without the possibility of appeal if they are participating in the tournament.
9. If someone intentionally decides not to play a map (remaining AFK for the entire duration), they will be penalized with a x0.75 multiplier to all their qualifier scores. This penalty will be reviewed on a case-by-case basis.

> ### Playoffs stage

Reviewer: just like how "compounds words" would be incorrect, compound words don't have a plural 's' inside them

```suggestion
### Playoff stage
```
